src/database/DataTypes/MasterList.js

```diff
@@ -87,13 +87,12 @@ export class MasterList extends Realm.Object {
    * @return {object} The matching storeTag programsettings field for the current store
    */
   getStoreTagObject(tags) {
-    const thisStoresTags = tags && tags.toLowerCase().split(/[\s,]+/);
     const storeTags = this.parsedProgramSettings && this.parsedProgramSettings.storeTags;
 
-    if (!(thisStoresTags && storeTags)) return null;
+    if (!(tags && tags.length && storeTags)) return null;
 
     const foundStoreTag = Object.keys(storeTags).find(
-      storeTag => thisStoresTags.indexOf(storeTag.toLowerCase()) >= 0
```

Comment:
method was taking a comma seperated string (`[store]tags`) - now accepting an array of tag strings (assuming they are passed lower case)

Reply:
Update the comment for param `tag`

Reply:
Thanks!


81734f1